meta-psa/recipes-security/psa-trusted-storage-linux/psa-trusted-storage-linux_1.0.0.bb

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+SUMMARY = "ARM Platform Security Architecture (PSA) Protected Storage Library"
+DEPENDS = ""
+HOMEPAGE = "https://github.com/ARMmbed/psa-storage"
+DESCRIPTION = "A Linux C library reference implementation of the PSA protected_storage.h API"
+SECTION = "libs"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = " \
+    file://${WORKDIR}/git/LICENSE;md5=302d50a6369f5f22efdb674db908167a \
+    file://${WORKDIR}/git/apache-2.0.txt;md5=3b83ef96387f14655fc854ddc3c6bd57 \
+    "
+
+SRC_URI = "git://git@github.com/armmbed/psa_trusted_storage_linux.git;protocol=ssh;nobranch=1"
+SRCREV = "2be73f8a50eca10b45f952798e2100afd00a99f2"
```

Comment:
```suggestion
SRCREV = "2be73f8a50eca10b45f952798e2100afd00a99f2"

PACKAGES += "${PN}-example"
```

Reply:
> `PACKAGES += "${PN}-example"`

I think you want `=+` or `_prepend` or `=.` here - the `-example` package should come before `${PN}` in `PACKAGES` so that it has first pick of the files in the sysroot.